src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PeopleRow } from '../PersonRow';
+import { useMemo } from 'react';
+import { SearchParamsForFilter } from '../../types/SearchParamsForFilter';
+import { SortParams } from '../../types/SortParams';
+import { Order } from '../../types/Order';
+import classNames from 'classnames';
+import { Param } from '../../types/Param';
+
+type Props = {
+  people: Person[];
+};
+
+function filterPeopleBySearchParams(
+  peopleList: Person[],
+  { queryParam, centuriesParam, peopleSexParam }: SearchParamsForFilter,
```

Comment:
if u have 3 or more arguments
```suggestion
  { 
    queryParam, 
    centuriesParam, 
    peopleSexParam 
  }: SearchParamsForFilter,
```